src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -157,28 +140,28 @@ public static void NonExistentAction_Throws_ActionNotSupportedException()
     [OuterLoop]
     public static void TimeoutTest_SendTimeout5Seconds()
     {
-        bool exceptionThrown = false;
+        BasicHttpBinding binding = new BasicHttpBinding();
+        binding.SendTimeout = TimeSpan.FromMilliseconds(5000);
+        ChannelFactory<IWcfService> factory = new ChannelFactory<IWcfService>(binding, new EndpointAddress(Endpoints.HttpBaseAddress_Basic));
+
+        Stopwatch watch = new Stopwatch();
         try
         {
-            BasicHttpBinding binding = new BasicHttpBinding();
-            binding.SendTimeout = TimeSpan.FromMilliseconds(5000);
-            ChannelFactory<IWcfService> factory = new ChannelFactory<IWcfService>(binding, new EndpointAddress(Endpoints.HttpBaseAddress_Basic));
-            IWcfService proxy = factory.CreateChannel();
-            proxy.EchoWithTimeout("Hello");
-        }
-        catch (TimeoutException)
-        {
-            exceptionThrown = true;
+            var exception = Assert.Throws<TimeoutException>(() =>
+            {
+                IWcfService proxy = factory.CreateChannel();
+                watch.Start();
+                proxy.EchoWithTimeout("Hello");
+            });
         }
-        catch (Exception ex)
+        finally
         {
-            Assert.True(false, String.Format("Unexpected exception caught: {0}", ex.ToString()));
+            watch.Stop();
         }
 
-        if (!exceptionThrown)
-        {
-            Assert.True(false, "Expected TimeoutException was not thrown nor was any other exception thrown.");
-        }
+        // want to assert that this completed in > 5 s as an upper bound since the SendTimeout is 5 sec
+        // (usual case is around 5001-5005 ms) 
+        Assert.InRange<long>(watch.ElapsedMilliseconds, 5000, 6000);
```

Comment:
I think the InRange so tightly restricted will cause this test to fail periodically in the labs when load is high.  Do we strictly care about the upper bound, as long as we know it is at least >= SendTimeout?